docs/src/ref/modeling.md

```diff
@@ -451,24 +451,36 @@ can instead be implemented as:
 ```
 
 ### Loading generated functions
-Before a function with a static annotation can be used, the [`load_generated_functions`](@ref) method must be called:
+Before a function with a static annotation can be used, the [`@load_generated_functions`](@ref) macro must be called:
 ```@docs
-load_generated_functions
+@load_generated_functions
 ```
 Typically, one call to this function, at the top level of a script, separates the definition of generative functions from the execution of inference code, e.g.:
 ```julia
-using Gen: load_generated_functions
+using Gen: @load_generated_functions
 
 # define generative functions and inference code
 ..
 
 # allow static generative functions defined above to be used
-load_generated_functions()
+@load_generated_functions
 
 # run inference code
 ..
 ```
 
+When static generative functions are defined in a Julia module, [`@load_generated_functions`](@ref) should be called after all static functions are defined:
```

Comment:
This is now no different than when you use it not within a module right? Is it necessary to add this?

Reply:
Hmm, I think it's good for clarity and as an example design pattern when you're writing library code which contains static generative functions. Importantly, as @alex-lew noted, once you call `@load_generated_functions` in a module, you no longer need to call `@load_generated_functions` in a script that uses the module. I don't think that's well communicated by the first usage example alone.

Reply:
Okay. In that case I think it could help to add a second code block showing the module being used. e.g.:

Then, when using this module, you don't need to call `@load_generated_functions` yourself to use the static generative functions defined in the module, e.g.:
```julia
using Gen
using MyModule: example_gen_fn
trace = simulate(example_gen_fn, ())
```

Reply:
Sounds good!